Packs/Base/TestPlaybooks/script-TestPyLogging.yml

```diff
@@ -0,0 +1,20 @@
+commonfields:
+  id: TestPyLogging
+  version: -1
+name: TestPyLogging
+script: |
+  import logging
+
+  root_logger = logging.getLogger()
+
+  logging.debug("this is a logging test")
```

Comment:
any value to print to different values here? `logging.info` for example

Reply:
don't see a need for it